__tests__/app.test.js

```diff
@@ -13,20 +13,98 @@ afterAll(() => {
 });
 
 describe("get/api/topics", () => {
-  test("Return topics", () => {
+  test("Return topics with the correct properties with 200", () => {
     return request(app)
-      .get('/api/topics')
+      .get("/api/topics")
       .expect(200)
       .then(({ body }) => {
         expect(body.topics).toHaveLength(3);
         body.topics.forEach((topic) => {
           expect(topic).toEqual(
             expect.objectContaining({
-              description: expect.any(String),              
+              description: expect.any(String),
               slug: expect.any(String),
             })
           );
         });
       });
   });
 });
+
+describe("get/api/articles", () => {
+  test("Return articles with 200 message, adding comment_count with the correct values", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toHaveLength(12);
+        body.articles.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              title: expect.any(String),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              article_img_url: expect.any(String),
+              article_id: expect.any(Number),
+              votes: expect.any(Number),
+              author: expect.any(String),
+              comment_count: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+});
+
+describe("get/api/articles/:articleid", () => {
```

Comment:
You haven't added in any error handling for this end point, make sure you are handling errors as you go along I wouldn't consider this end point complete until all the errors have been handled